index.html

```diff
@@ -1,205 +1,283 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>1 Portafolio</title>
-    <link rel="stylesheet" href="css/style.css">
-    <script src="https://kit.fontawesome.com/84fe87cdc9.js" crossorigin="anonymous"></script>
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
-</head>
-<body>
-      <section class="header">
-      <header>
-        <nav class="navflex">
-            <div class="mylogo">My Logo</div>
-            <span class="mobilmenu"><i class="fa-solid fa-bars"></i></span>       
-             <ul class="menudesktop">
-                <li class="md"><a href="">Portafoliio</a> </li>
-                <li class="md"><a href="">About</a> </li>
-                <li class="md"><a href="">Contac</a> </li>
-             </ul>    
+    <link rel="stylesheet" href="css/style.css" />
+    <script
+      src="https://kit.fontawesome.com/84fe87cdc9.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <section class="header">
+      <header class="header2">
+        <div class="menu_cont">
+          <div class="mylogo">
+            <a class="color-logo" href="#">My Logo</a>
+          </div>
+          <nav class="headbar">
+            <div class="direction">
+              <ul class="mainmenu">
+                <li class="margin-menu">
+                  <a class="menu-link" href="#portfolio">Portfolio</a>
+                </li>
+                <li class="margin-menu">
+                  <a class="menu-link" href="#about">About</a>
+                </li>
+                <li class="margin-menu">
+                  <a class="menu-link" href="#contact">Contact</a>
+                </li>
```

Comment:
- [ ] Please kindly ensure that when any of these menu items is clicked, the correct part of the page is displayed as stated in the requirement. 👍 

<img width="593" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/231130813-d62b29db-7b93-400e-b86a-5778dc48b27b.png">